terraform-environments/baremetal-hosts.tf

```diff
@@ -0,0 +1,39 @@
+# data "template_file" "user_data" {
+#   template = file("../host_user_data.sh")
+# }
+
+data "template_file" "user_data" {
+  template = <<-EOT
```

Comment:
ideally you should factor this out into a separate script (which we can then lint and then validate - and here we would just be calling it